resources/dicts/patrols/general/border.json

```diff
@@ -1936,8 +1936,8 @@
                 "other_clan_rep": 2,
                 "relationships":
                 {
```

Comment:
looks like this relationships block is missing it's brackets to make it a list

Reply:
added!